homework-submission/week4-homework/src/components/Checkbox.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+class Checkbox extends React.Component {
+  render() {
+    let done = this.props.done;
+    return (
+      <button onClick={this.props.onClick} className="checkbox">
```

Comment:
Usually when we see this line of code we say that it's mentioned to an event Object passed by an argument field, That I didn't mention before but it's good to know.

So if you want to pass something pass it what may make more sense for you in the future and for others as well.